src/App.js

```diff
@@ -17,6 +20,14 @@ const App = () => (
           <Route exact path="/">
             <Redirect to="/map" />
           </Route>
+          <Route exact path="/about/project">
+            <Header />
```

Comment:
Is there a way we can render these routes while using the existing MainPage --> DesktopLayout structure? This works for desktop, but how would your current solution work when we want to add support for these pages on mobile?